src/types/core/ClientOptions.ts

```diff
@@ -3,6 +3,11 @@ import { GuardFunction } from "../public/GuardFunction";
 import { LoadClass } from "./LoadClass";
 
 export interface ClientOptions extends DiscordJSClientOptions {
+  /**
+   * Specifiy bot id (added for multiple bot support)
+   */
+  botid?: string;
```

Comment:
Are you sure this should be optional? Isn't it required now?

Reply:
yes that would be better too.

Reply:
added